docs/administration/vulnerability-scanning/import-vulnerability-data.md

```diff
@@ -62,4 +62,4 @@ docker exec harbor-db /bin/sh -c  "pg_dump -U postgres -c" > all.sql
 
 ## Rescan the Images
 
-After importing the data, trigger the scanning process in the Harbor interface. For information about running a scan, see [Scan All Images](scan-all-images.md).
+After importing the data, trigger the scanning process in the Harbor interface. For information about running a scan, see [Scan All Images](scan-all-artifacts).
```

Comment:
Can we change the link to be Artifacts as well?

Reply:
we sure can! :)